build/karma.conf.js

```diff
@@ -55,7 +55,7 @@ module.exports = function(config) {
 
     logLevel: 'WARN',
 
-    frameworks: ['jasmine', 'browserify'],
+    frameworks: ['jasmine-jquery', 'jasmine', 'browserify'],
```

Comment:
In order to make this work: 
https://github.com/kubernetes/dashboard/pull/358/files#diff-1fffe0e288e12a130547d3df522ebad5R96

This plugin is required. Without that even if I directly append `document.body` with fixture element, still offsetWidth of elements will not be updated. Will be always 0. Additionally without that `find` function does not work correctly. It does not find elements by class.

I couldn't make it work without this plugin.